addons/social_share/models/res_partner.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models
+
+
+class Partner(models.Model):
+    _inherit = 'res.partner'
+
+    def _snshare_allowed_fields(self):
+        return ['name', 'phone', 'mobile', 'email', 'country_id']
```

Comment:
Shoudln't we also include country_id.name then, if we make the path validation on a "rendering model" basis ?